test/integration/034_redshift_test/test_backup_table_option.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from test.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestBackupTableOption(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return 'backup_table_option_034'
+
+    @staticmethod
+    def dir(path):
+        return os.path.normpath(path)
+
+    @property
+    def models(self):
+        return self.dir("models_backup")
+
+    @property
+    def project_config(self):
+        return {
+            'config-version': 2,
+            'data-paths': [self.dir('seed')],
+            'seeds': {
+                'quote_columns': False,
+            }
+        }
```

Comment:
> Everything seemed straightforward to me besides "and once via `dbt_project.yml`". Do you mean to update that within the test and re-run dbt? Is there a good example of doing that somewhere else that I could look at?

Yes, all I meant here was creating another test case, with only one difference:
```python
    @property
    def project_config(self):
        return {
            'config-version': 2,
            'data-paths': [self.dir('seed')],
            'seeds': {
                'quote_columns': False,
            },
            'models': {
                'backup': False
            }
        }
```
Since this should affect `model_backup_undefined` by causing it to include `BACKUP NO`. I see you've defined the `project_config_append` method, was that to accomplish this?